### PR TITLE
docs: add rxweb reactive form validation library to the new list of Reactive Forms

### PR DESCRIPTION
**Pull request by sholkajadav11**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by StephenFluin — CHANGES_REQUESTED** [review]
The tool looks valuable.

Can you instead put it in the "Data Libraries" category, and make sure your JSON is valid?

**Comment by sholkajadav11** [comment]
@StephenFluin I have done the changes as per your suggestion.

**Review by StephenFluin — APPROVED** [review]
Final content looks good, but you'll need to squash your commits.

https://twitter.com/stephenfluin/status/1009904095073718275?lang=en

**Comment by sholkajadav11** [comment]
@StephenFluin I squashed the commit, Please check.

**Comment by StephenFluin** [comment]
@sholkajadav11 I'm seeing 4 commits right now in the PR. There should only be one.

**Comment by sholkajadav11** [comment]
@StephenFluin Please check now.

**Review by StephenFluin — APPROVED** [review]

**Comment by sholkajadav11** [comment]
@StephenFluin @kapunahelewong I would like to know when this will be merged?

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -110,6 +110,12 @@
       "Data Libraries": {
         "order": 3,
         "resources": {
+           "RxWebReactiveFormValidators":{
+            "desc":"RxWeb Reactive Form Validators provides all type of complex, conditional, cross field and dynamic validation on validator based reactive forms, model based Reactive Form and Template Driven Form.",
```
Comment (kapunahelewong) [inline aio/content/marketing/resources.json]:
```suggestion
            "desc":"RxWeb Reactive Form Validators provides all types of complex, conditional, cross field, and dynamic validation on validator-based reactive forms, model-based reactive forms, and template driven forms.",
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Hi @sholkajadav11! Thanks for the tag - it really is super helpful for me. I've included some minor copy edits. When you're all ready (or if you have any questions) feel free to tag me. (By the way, ultimately, you'll want just one commit. Here are [my notes on the process](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) if you'd like them.)

EDIT: Oh, you already know the squashing part. 😊

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (sholkajadav11) [reply aio/content/marketing/resources.json]: @kapunahelewong Thanks for the edits in the description :) . I have squashed the commit. Please let me know if anything I have to do.

**Comment by sholkajadav11** [comment]
@kapunahelewong I have squashed the commit, Please check.

File: aio/content/marketing/resources.json [hunk 2]
```diff
@@ -110,6 +110,12 @@
       "Data Libraries": {
         "order": 3,
         "resources": {
+           "RxWebReactiveFormValidators":{
```
Comment (kapunahelewong) [inline aio/content/marketing/resources.json]: Oh, I missed this earlier. Could you fix the formatting? And [amend the commit message](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-amend-a-commit)?

For the commit message, can you try something like this?

>docs: add rxweb reactive form validation library to resources

This means you'll remove the body. This part:

```
Update resources.json

docs: add @rxweb/reactive-form-validators info

Update resources.json

docs: add @rxweb/reactive-form-validators info
```

Please tag me if it gets tricky and I can help. Thanks again, @sholkajadav11.

**Comment by sholkajadav11** [comment]
Thank you @kapunahelewong for quick response, I fixed the formatting and amend the commit message. Please check :) .

**Review by kapunahelewong — APPROVED** [review]

**Comment by kapunahelewong** [comment]
Looks good! Can you [rebase](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase)? That failure looks unrelated to your change, as maybe something that was fixed since you submitted the PR. Notice that there is a 159 commit difference between your fork and angular/angular master, so I think a rebase will fix it. If it doesn't but you're all up-to-date, I can then reach out for merge assistance.  Thanks, @sholkajadav11 🌺

**Comment by sholkajadav11** [comment]
@kapunahelewong I did mistake, Please guide me how I can correct.

**Comment by kapunahelewong** [comment]
I think I got it from my end, @sholkajadav11 🌈. Would you like to review to make sure everything is how you intend it? To do it I had to check it out locally on my machine and push up. I backed up what you had just in case, but can you check after me? Thank you!!!

You're so close!! 🌺

**Comment by sholkajadav11** [comment]
All Good @kapunahelewong , Thank you very much. You are so kind. Once again thank you @kapunahelewong :).

**Comment by kapunahelewong** [comment]
🎉🌈🌸😊

**Comment by sholkajadav11** [comment]
Thank you @kapunahelewong for your support.

**Comment by kapunahelewong** [comment]
It was great to see this get merged! 🌟😊

**Pull request closed without merging**